packages/cubejs-redshift-driver/src/RedshiftDriver.ts

```diff
@@ -5,14 +5,24 @@ import crypto from 'crypto';
 import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
 import { S3, GetObjectCommand } from '@aws-sdk/client-s3';
 
-interface RedshiftDriverExportAWS {
+interface RedshiftDriverExportRequiredAWS {
   bucketType: 's3',
   bucketName: string,
-  keyId: string,
-  secretKey: string,
   region: string,
 }
 
+interface RedshiftDriverExportArnAWS extends RedshiftDriverExportRequiredAWS{
+  unloadArn?: string,
+}
+
+interface RedshiftDriverExportKeySecretAWS extends RedshiftDriverExportRequiredAWS{
```

Comment:
Hmm, looks to me like we always need key/secret, because that's what Cube needs to use to download the files. How about:

```
interface RedshiftDriverExportAWS {
  bucketType: 's3',
  bucketName: string,
  keyId: string,
  secretKey: string,
  region: string,
  unloadArn?: string,
}
```

Reply:
Same as the above comment, if the task/container/lambda/etc are assigned a role with permissions, the underlying SDK will be able to make the request.

These keys are helpful in development, where the dev machine probably does not have an `iam_role` with permissions.